.gitignore

```diff
@@ -43,3 +43,4 @@ vendor/bundle/
 coverage
 /reports/
 !/reports/README.md
+bin/
```

Comment:
I don't see why we can't share the binstubs. I would not ignore it.

Reply:
The objective is to keep the number of files in the project to a minimum. Less complexity to take.
Also, most people will not use this as they are using zeus.
all you need to do is to run the commands above to generate them.

Reply:
as you want. I might migrate to spring as well, to be stick to a single solution for the team.